pkg/defaults/defaults.go

```diff
@@ -256,37 +260,62 @@ func stepConfigsForBuild(config *api.ReleaseBuildConfiguration, jobSpec *api.Job
 		config.InputConfiguration.BaseRPMImages[alias] = target
 	}
 
-	buildSteps = append(buildSteps, api.StepConfiguration{SourceStepConfiguration: &api.SourceStepConfiguration{
-		From:      api.PipelineImageStreamTagReferenceRoot,
-		To:        api.PipelineImageStreamTagReferenceSource,
-		PathAlias: config.CanonicalGoRepository,
-		ClonerefsImage: api.ImageStreamTagReference{
-			Cluster:   "https://api.ci.openshift.org",
-			Namespace: "ci",
-			Name:      "clonerefs",
-			Tag:       "latest",
-		},
-		ClonerefsPath: "/clonerefs",
-	}})
+	if target := config.InputConfiguration.BuildRootImage; target != nil {
+		if isTagRef := target.ImageStreamTagReference; isTagRef != nil {
+			if isTagRef.Namespace == "" {
```

Comment:
Everything that takes the `isTagRef` and creates the `InputImageTagStepConfiguration` is duplicated here and for the `TestBaseImage` -- factor this to share logic.